lib/widgets/drive/compare.coffee

```diff
@@ -0,0 +1,40 @@
+_ = require('lodash')
+
+containsDeep = (array, item) ->
+	return _.any(_.map(array, _.partial(_.isEqual, item)))
+
+differenceDeep = (x, y) ->
+	return _.filter(x, _.partial(_.negate(containsDeep), y))
+
+createDiffOperation = (type, element) ->
+	return {
+		type: type
+		drive: element
+	}
+
+###*
+# @summary Detect changes regarding drives between different time intervals
+# @name compare
```

Comment:
This is not necessary neither. It was needed in the SDK to workaround a markdown generation issue.